x-pack/plugins/actions/server/lib/mustache_renderer.ts

```diff
@@ -56,11 +56,41 @@ export function renderMustacheObject<Params>(params: Params, variables: Variable
 
 // return variables cloned, with a toString() added to objects
 function augmentObjectVariables(variables: Variables): Variables {
+  // convert context variables with '.' in the name to objects
+  if (variables.context) {
```

Comment:
even though `context` is where alerting stores all it's variables for access by actions, the mustache renderer doesn't really treat `context` specially, and I don't think it should.

We also probably want to do this AFTER the `JSON.parse(JSON.stringify())` below, which is a sanitizer / validator.  You do want to call it BEFORE `addToStringDeep()`, as we want the newly created objects to get the extra `toString()` added to them.

I don't think we really need to deal with `context` at all?  Just pass in `variables`?

Reply:
Resolved in this commit 00406ed0c744dff5775a571da58b0ba2d25295a6